modules/tinymce/src/core/test/ts/browser/html/DomParserTest.ts

```diff
@@ -1443,8 +1443,8 @@ describe('browser.tinymce.core.html.DomParserTest', () => {
         context('Transparent elements', () => {
           const getTransparentElements = (schema: Schema) => Arr.unique(Arr.map(Obj.keys(schema.getTransparentElements()), (s) => s.toLowerCase()));
 
-          const testSplitInvalidBlocksOut = (testCase: { input: string; expected: string }) => {
-            const parser = DomParser(scenario.settings);
+          const testTransparentElementsParsing = (testCase: { input: string; expected: string; domParserSettings?: DomParserSettings }) => {
+            const parser = DomParser({ ...scenario.settings, ...testCase.domParserSettings ?? {}});
```

Comment:
Object spread operator on nullables returns an empty object
```suggestion
            const parser = DomParser({ ...scenario.settings, ...testCase.domParserSettings });
```

Reply:
Ahh, cool didn't know that. Thanks.